esp/esp/program/models/__init__.py

```diff
@@ -1865,6 +1865,22 @@ def __unicode__(self):
         else:
             return self.name
 
+class PhaseZeroRecord(models.Model):
+
+    class Meta:
+        app_label = 'program'
+
+    def __unicode__(self):
+        return self.lottery_number
+
+    user = AjaxForeignKey(ESPUser)
+    program = models.ForeignKey(Program, blank=True, null=True)
+    phase_zero_time = models.DateTimeField(default=timezone.now)
+    lottery_number = models.CharField(max_length=6)
```

Comment:
Originally we planned to have the lottery codes be hex values (hence the CharField), but then the plan changed. We'll move over to IntegerField.